src/pages/Trainee/TraineeList.jsx

```diff
@@ -30,6 +31,51 @@ class TraineeList extends Component {
     };
   }
 
+  componentDidMount() {
+    const { data: { subscribeToMore } } = this.props;
+    subscribeToMore({
+      document: UPDATE_TRAINEE_SUB,
+      updateQuery: (prev, { subscriptionData }) => {
+        if (!subscriptionData) return prev;
+        const { getAllTrainee: { records } } = prev;
```

Comment:
const { getAllTrainee: { records } } = prev;
can be written as..
const { getAllTrainee: { records } ={}} = prev||{};